src/components/containers/CreateToken.container.tsx

```diff
@@ -56,23 +60,34 @@ const deleteToken = (
   onDeleteApiToken({ token, apiToken })
 }
 
-const getApiTokenByNetwork = (apiTokens: ReadonlyArray<string>, network: Network): ReadonlyArray<string> =>
+const getApiTokenByNetwork = (network: Network) => (apiTokens: ReadonlyArray<string>): ReadonlyArray<string> =>
   apiTokens.filter((apiToken: string) => parseJwt(apiToken).network === network)
 
+const capitalize = ([first, ...rest]: string) => first.toUpperCase() + rest.join('').toLowerCase()
+const isActiveToggleNetwork = () => isActive('toggle-network', getCurrentActiveFeatures({ initialFeatures }))
```

Comment:
Should we use Features.ToggleNetwork here?

Reply:
yes, because it fails with the communication with the Frost api. We have to avoid sending the new property `network`

do you have any better solution?

Reply:
Sorry I mean instead of 'toggle-network'